spec/abci/abci++_methods.md

```diff
@@ -452,7 +362,11 @@ title: Methods
       that the `RequestPrepareProposal.max_tx_bytes` limit is respected by those transactions
       returned in `ResponsePrepareProposal.txs` .
     * As a result of executing the prepared proposal, the Application may produce block events or transaction events.
-      The Application must keep those events until a block is decided. It will then forward the events to the `BeginBlock-DeliverTx-EndBlock` functions depending on where each event should be placed, thereby returning the events to CometBFT.
+      The Application must keep those events until a block is decided and then pass them on to CometBFT via
```

Comment:
This <tab> here makes GitHub to format the remaining of this item list as a code. The comments, in particular, are rendered :D